css/main.css

```diff
@@ -34,6 +34,60 @@ label.dynamic-info {
 	font-style: italic;
 }
 
+input[type='number'] {
+	max-width: 7em;
+	/* Remove spinner controls from Firefox */
+	-moz-appearance: textfield;
+}
+/* Re-applies the controls on :hover and :focus
+to get the same behavior as on chrom* */
+input[type='number']:hover,
+input[type='number']:focus {
+	-moz-appearance: number-input;
+}
+
+/* Enables the first input's help text to pop
+over the lasts of this group.
+Also keeps related inputs together. */
+fieldset.no-space-wrap {
+	white-space: nowrap;
+	display: inline-block;
+}
+/* Reset fieldset style */
+fieldset {
+	border: 0;
+  padding: 0.01em 0 0 0;
+  margin: 0;
+  min-width: 0;
+}
+
+label[for='salaire'] {
+	position: relative;
+}
+
+label[for='salaire']:hover ~ * {
+	visibility: hidden;
+}
+label[for='salaire']:hover .input-help {
+	visibility: visible;
+	opacity: 1;
```

Comment:
The rule block `label[for='salaire']:hover ~ * ` will also set visible hidden to `input-help`, hence the need to set it visible. Then the opacity enables the transition. The transition: visibility seems useless though, I don't remember a good reason for it.